tests/data/configs_mmtrack/selsa_faster_rcnn_r101_dc5_1x.py

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Seems that we do not need to add copyright in config files, can we ignore the `configs` folder

Reply:
Done